### PR TITLE
Sprint1 - FULL

### PR DESCRIPTION
**Pull request by Bachir-Ndiaye**
=> Merged all classmates work
=> Added some CSS on diary page to fit the figma model
=> made a dataset for the presentation

### CODE REVIEW

**Comment by Bachir-Ndiaye** [comment]
Resolved conflicts

File: yarn.lock [hunk 1]
```diff
@@ -5286,7 +5289,20 @@ postcss-discard-empty@^5.0.1:
...
+  integrity sha512-/+Z1RAmssdiSLgIZwnJHwBMnlABPgF7giYzTN2NOfr9D21IJZ4mQC1R2miwp80zno9M4zMD/umGI8cR+2EL5zw==
+  dependencies:
+    cosmiconfig "^7.0.0"
+    klona "^2.0.4"
+    semver "^7.3.4"
+
+postcss-merge-longhand@^5.0.1:
+=======
 postcss-discard-overridden@^5.0.1:
+>>>>>>> dev
```
Comment (aldaitzwild) [inline yarn.lock:5305]: Unsolved conflicts

File: yarn.lock [hunk 2]
```diff
@@ -924,6 +924,8 @@
     "@nodelib/fs.scandir" "2.1.4"
     fastq "^1.6.0"
 
+<<<<<<< HEAD
+=======
```
Comment (aldaitzwild) [inline yarn.lock:928]: Unsolved conflicts

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
Conflicts with dev

**Pull request closed without merging**